quint/test/runtime/compile.test.ts

```diff
@@ -1053,5 +1054,33 @@ describe('incremental compilation', () => {
       const computable = context.evaluationState?.context.get(kindName('callable', def!.id))!
       assertComputableAsString(computable, '3')
     })
+
+    it('uses the oringial modules to resolve names', () => {
```

Comment:
```suggestion
    it('uses the original modules to resolve names', () => {
```

Reply:
Unless I misunderstand, this test name seems to reflect an implementation detail rather than the behavior that is actually being tested, which is more like: "non-exported imports are not visible in subsequent importing modules".